src/main/app/claims/models/response/responseCommon.ts

```diff
@@ -16,6 +16,8 @@ export interface ResponseCommon {
   defendant: Party
   moreTimeNeeded?: YesNoOption
   freeMediation?: YesNoOption
+  mediationPhoneNumber?: String
```

Comment:
We probably want to use the standard `string` TS type rather than the JS one

Reply:
good spot, ofcourse, that was a mistake. I have corrected it now.